test/infrastructure/docker/exp/controllers/alias.go

```diff
@@ -37,10 +38,11 @@ type DockerMachinePoolReconciler struct {
 }
```

Comment:
```suggestion
	Tracker          *remote.ClusterCacheTracker
}
```
You have to keep it here. Otherwise it won't compile

Reply:
I think there was a misunderstanding. I didn't mean that you have to move it to the method signature.
I just meant that you have to add 
```
		Tracker:          tracker,
```
in l.46